distribution/docs/src/main/resources/_contents/_data/topic-attributes-table-contents.adoc

```diff
@@ -11,22 +11,22 @@
 
 |topic.category
 |A category code from a given vocabulary.
-|String
-|should be restricted to the values in a given vocabulary
+|List of Strings
+|< 1024 characters per entry; should be restricted to the values in a given vocabulary
 |
 
 |topic.keyword
 |One or more keywords describing the subject matter of the
 metacard or resource.
-|String
-|< 1024 characters
+|List of Strings
+|< 1024 characters per entry
 |
 
 |topic.vocabulary
 |An identifier of a controlled vocabulary from which the
 topic category is derived.
-|String
-|< 1024 characters
+|List of Strings
+|< 1024 characters per entry
```

Comment:
@jlcsmith how should we constrain this? I am thinking controlled vocabularies should be identified by a URI within the system. This would make it easier to build robust validation for these fields down the line.